### ISSUE TITLE
Variables in media queries do not resolve

### ISSUE THREAD

**Issue by SimplGy** [sass/sass#341]
If I put this into sass, it does not resolve the $bigScreen variable and does not throw any kind of error.

Seems like this should work. Should it not?

```
$bigScreen: 1280px

@media (min-width: $bigScreen)
  .panel
    width: 50%
```

http://sass-lang.com/try.html

**Comment by chriseppstein** [comment]
This is already fixed in sass 3.2. `gem install sass --pre`

**Comment by SimplGy** [comment]
Awesome, thanks Chris!
